### PR TITLE
Gauge integration

### PR DESCRIPTION
**Pull request by JackCollett**
PR for Gauge Component and Buttons, all fully functional with existing code on the 'counter' and increment/decrement code.

### CODE REVIEW

**Review by F-Gulmez — DISMISSED** [review]

File: Frontend/src/components/Gague.jsx [hunk 1]
```diff
@@ -1,7 +1,38 @@
 import { useState } from "react";
 import Buttons from './Buttons'
+import * as React from 'react';
+import { Gauge } from '@mui/x-charts/Gauge';
```
Comment (tomgame984) [inline Frontend/src/components/Gague.jsx]: Could this be added into the import below?
```suggestion
```

File: Frontend/src/components/Gague.jsx [hunk 2]
```diff
@@ -1,7 +1,38 @@
...
+import {
```
Comment (tomgame984) [inline Frontend/src/components/Gague.jsx:3]:
```suggestion
import { Gauge,
```

**Review by tomgame984 — CHANGES_REQUESTED** [review]
Small change recommended by SonarQub to improve maintainability.

File: Frontend/src/components/Gague.jsx (hunk 2 above)
Comment (JackCollett) [reply Frontend/src/components/Gague.jsx:3]: Didn't spot that, Thank you! I was also able to merge the React libraries as well.

**Pull request closed without merging**